pkg/cmd/pulumi/util.go

```diff
@@ -132,9 +132,20 @@ func loginToCloud(
 	project *workspace.Project,
 	insecure bool,
 	opts display.Options,
+) (backend.Backend, error) {
+	return loginToCloudInternal(ctx, cloudURL, project, insecure, false /* setCurrent */, opts)
+}
+
+func loginToCloudInternal(
```

Comment:
LoginToCloud is deleted on master, just use the new login manager system it already supports setCurrent

Reply:
Nice!